pyproject.toml

```diff
@@ -0,0 +1,167 @@
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[project]
+name = "dallinger"
+version = "9.11.0a1"
+maintainers = [
+  {name = "Jordan Suchow", email = "suchow@berkeley.edu"}
```

Comment:
Let's take this opportunity to update the maintainers list, perhaps? It could for example be Jordan, Frank, myself, and then Jazkarta?

Reply:
I will add the both of us for the moment as  the Jazkarta maintainer email is not clear yet.

Reply:
Sounds good

Reply:
Done.